pkg/autodiscovery/autoconfig.go

```diff
@@ -351,6 +360,38 @@ func (ac *AutoConfig) AddLoader(loader check.Loader) {
 	ac.loaders = append(ac.loaders, loader)
 }
 
+func decryptConfig(conf integration.Config) (integration.Config, error) {
+	var err error
+
+	// init_config
+	conf.InitConfig, err = encryptedsecret.Decrypt(conf.InitConfig)
+	if err != nil {
+		return conf, fmt.Errorf("Dropping conf for '%s', error while decrypting passwords in 'init_config': %s", conf.Name, err)
```

Comment:
again, to keep the naming consistent, I would say `secrets` instead of `passwords` here and in all the other messages of this file